code/modules/assembly/logic/math.dm

```diff
@@ -28,9 +28,11 @@ var/global/list/math_circuit_operations_list = list("ADD", "SUBTRACT", "MULTIPLY
 	var/list/obj/item/device/assembly/values = list() //List of constants (numbers) or variables (assemblies). All assemblies in this list have a string associated with them, which tells this circuit which of the assembly's values to use
 	var/operation = "ADD"
 
+	//Allow devices to read this circiut's result. First parameter (variable name, which is "null" here) isn't important - the functions are overwritten
 	accessible_values = list("Result" = "null;number",\
-		"Operation" = "operation;text") //Allow devices to read this circiut's result. First parameter (variable name, which is "null" here) isn't important - the functions are overwritten
-
+		"Operation" = "operation;text",\
+		"Variables" = "null;text")
```

Comment:
Check if you can do the values as string additions, that way you can use the define instead.